src/biclustering_impl.h

```diff
@@ -225,11 +131,10 @@ class ConvexBiClustering {
     }
 
     nzeros_row = v_row_zeros.sum();
+    ClustRVizLogger::debug("zeros_row");
```

Comment:
Should there be more info here?

Reply:
This was a progress report print. I'll remove